tests/api_tests/features/api_tests.feature

```diff
@@ -0,0 +1,95 @@
+@api
+Feature: Newsfeed API tests
+
+  Scenario: Check microservice status
+    When get request to /status/ is received
+    Then response status should be 200
+    And response should contain "OK" status
+
+  Scenario: Verify documentation
+    When get request to /docs/ is received
+    Then response status should be 200
+    And response should contain documentation
+
+  Scenario: Get events
+    When get request to /newsfeed/123/events/ is received
+    Then response status should be 200
+    And response should contain results list
+
+  Scenario: Add events
+    When post request to /newsfeed/123/events/ is received with body:
+      | field_1   | some_data  |
+      | field_2   | other_data |
+      | field_etc | etc_data   |
+    Then response status should be 202
+    And response should include the event
+
+  Scenario: Delete event
+    Given post request to /newsfeed/123/events/ is received with body:
+      | field_1   | some_data  |
+      | field_2   | other_data |
+      | field_etc | etc_data   |
+    When delete request to /newsfeed/123/events/4567/ is received
+    Then response status should be 204
+    And this event should be absent
+
+  Scenario: Get subscriptions
+    When get request to /newsfeed/123/subscriptions/ is received
+    Then response status should be 200
+    And response should contain results list
+
+  Scenario: Add subscriptions
+    When post request to /newsfeed/123/subscriptions/ is received with body:
+      | to_newsfeed_id | 124 |
+    Then response status should be 201
+    And response should include the subscription
+
+  Scenario: Delete subscription
+    Given post request to /newsfeed/123/subscriptions/ is received with body:
+      | to_newsfeed_id | 124 |
+    When delete request to /newsfeed/123/subscriptions/4567/ is received
+    Then response status should be 204 (500-???)
```

Comment:
`500` error happens when we try to delete absent subscription. API should return 404 in such case.